returnn/util/file_cache.py

```diff
@@ -327,6 +327,20 @@ def _check_existing_copied_file_maybe_cleanup(src_filename: str, dst_filename: s
         return True
 
 
+def _copy_with_prealloc(src: str, dst: str):
+    """
+    copies from src to dst preallocating the disk space at dst before copying
```

Comment:
There should always be an empty line after the description and the params in the docstring.

Or you leave the params away anyway. I think in this case the params are not really needed to be described. If you think`src` and `dst` are too ambiguous, then call them `source_file` and `destination_file` instead. That makes the param docstring obsolete.